ecoindex_cli/cli/app.py

```diff
@@ -82,9 +87,14 @@ def analyze(
             raise Exit(code=1)
 
         domain = urlparse(next(iter(urls))).netloc
+        if urls_file:
+            domain = urls_file.split("/")[-1]
         write_urls_to_file(domain=domain, urls=urls)
         secho(f"📁️ Urls recorded in file `/tmp/ecoindex-cli/input/{domain}.csv`")
 
+        output_log = f"/tmp/ecoindex-cli/output/{domain}.log"
+        log = logger(str(output_log))
```

Comment:
```suggestion
        log = Logger(filename=output_log)
```

For a class, it is better to respect [CapsWord](https://www.python.org/dev/peps/pep-0008/#class-names) convention

Reply:
I also take advantage of named parameters which is a good practice